source/Module/Rule.VsphereAcceptanceLevel/Convert/VsphereAcceptanceLevelRule.Convert.psm1

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+using module .\..\..\Common\Common.psm1
+using module .\..\VsphereAcceptanceLevelRule.psm1
+
+$exclude = @($MyInvocation.MyCommand.Name,'Template.*.txt')
+$supportFileList = Get-ChildItem -Path $PSScriptRoot -Exclude $exclude
+foreach ($supportFile in $supportFileList)
+{
+    Write-Verbose "Loading $($supportFile.FullName)"
+    . $supportFile.FullName
+}
+# Header
```

Comment:
Does it make sense to use #region header here?

Reply:
Removed